lib/google_scraping/dashboard/schemas/keyword_url.ex

```diff
@@ -0,0 +1,30 @@
+defmodule GoogleScraping.Dashboard.Schemas.KeywordUrl do
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
+  @fields [
+    :url,
+    :is_ad,
+    :user_id,
+    :keyword_id
+  ]
+
+  schema "keyword_urls" do
+    field :url, :string
+    field :is_ad, :boolean
+
+    belongs_to :user, GoogleScraping.Accounts.Schemas.User
```

Comment:
Do we need it? because from the `keyword` we know the `user`

Reply:
Without setting the user here, we would need to do joins through keywords tables to load keyword URLs for the specific user.

Sometimes denormalization to speed things up is an acceptable solution. What do you think?